.github/workflows/test-software.eessi.io.yml

```diff
@@ -56,6 +72,24 @@ jobs:
                   if [[ ${ec} -ne 0 ]]; then echo "missing installations found for ${easystack_file}!" >&2; exit ${ec}; fi
               done
 
+              # now check the accelerator builds for this CPU target
+              accelerators=$(echo "${EESSI_ACCELERATOR_TARGETS}" | yq ".${EESSI_SOFTWARE_SUBDIR_OVERRIDE}[]")
+              if [ -z ${accelerators} ]; then
+                  echo "no accelerator targets defined for ${EESSI_SOFTWARE_SUBDIR_OVERRIDE}"
+              else
+                  for accel in ${accelerators}; do
+                      module use ${EESSI_SOFTWARE_PATH}/accel/${accel}/modules/all
+                      echo "checking missing installations for accelerator ${accel} using modulepath: ${MODULEPATH}"
+                      for easystack_file in $(ls easystacks/software.eessi.io/${{matrix.EESSI_VERSION}}/accel/nvidia/eessi-${{matrix.EESSI_VERSION}}-eb-*.yml); do
```

Comment:
You should use `${accel} here rather than hardcoding `/accel/nvidia`?

Reply:
Done in https://github.com/EESSI/software-layer/pull/753/commits/eaf656a9d38e1af007484589f4fee615c99d5778 by using `dirname ${accel}`.

Reply:
Hardcoding `accel/` is fine, but not the `nvidia` part, indeed